### PR TITLE
add direction rtl to footer container story

### LINKED ISSUE #2197: Add container story for FooterContainer
~blocked: https://github.com/bbc/psammead/issues/757 and https://github.com/bbc/psammead/issues/770~

**Is your feature request related to a problem? Please describe.**
As we have moved components over to the Psammead component library, we have few items in our Simorgh Storybook: https://bbc.github.io/simorgh/

We should include our container components in the Simorgh storybook.

**Describe the solution you'd like**
Add story to Footer Container (on storybook)

- [x] What would be ideal, would be to add a drop-down that could switch between services and populate these with the correct view. e.g. selecting Persian from the drop-down when looking at the BrandContainer will show you the News | Persian brand.
- [x] The containers relating to RTL services, (e.g. Persian and in future, other RTL services) should have the `dir="rtl"` attribute passed through to Storybook. Could be added via a Decorator, that gets the value of `dir` from the Service Context. 

See the implementation of this for the Psammead sitewide links component - https://bbc.github.io/psammead/?path=/story/components-sitewidelinks--default When you select `Persian` from the `language` dropdown, it loads links in the right language and adds the correct `dir` value too.

Footer storybook container: https://github.com/bbc/simorgh/blob/latest/src/app/containers/Footer/index.stories.jsx

**Describe alternatives you've considered**

**Testing notes**
Regression of the FooterContainer

Dev insight: Cypress tests for Storybook may need to be updated, if so pair with test. 

**Additional context**
We shouldn't need stories for all of our containers (e.g. Blocks, or Fragments) since these are for handling our Optimo data.

### PR DESCRIPTION
**Pull request by thekp**
Resolves #2197
blocked by: https://github.com/bbc/psammead/pull/758 and https://github.com/bbc/psammead/issues/770

**Overall change:** _Update footer container storybook to have a dropdown of services and have RTL direction for required services._

**Code changes:**

- _added storybook-helper package_
- _added knobs for the footer storybook_

**Test notes:**
- _build branch locally_
- _check story book for footer, that it works as expected_
- _no new cypress tests required_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [n/a] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

**Comment by thekp** [comment]
At the moment, the dropdown of services is not restricted to existing services. However, this issue will cover this: https://github.com/bbc/psammead/issues/757

File: src/app/containers/Footer/index.stories.jsx [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
-    stories.add(service, () => (
-      <ServiceContextProvider service={service}>
-        <Footer />
-      </ServiceContextProvider>
-    ));
-  });
+storiesOf('Footer', module)
+  .addDecorator(withKnobs)
+  .add(
+    'testing',
```
Comment (ryanmccombe) [inline src/app/containers/Footer/index.stories.jsx]: I guess this label was a placeholder?

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/Footer/index.stories.jsx (hunk 1 above)
Comment (thekp) [reply src/app/containers/Footer/index.stories.jsx]: nope, miss named there... will update shortly :)

**Review by dr3 — APPROVED** [review]

**Comment by PriyaKR** [comment]
@thekp Looks good to me.(Can we make the container title as `Footer container` instead of `footer` in the storybook just so its consistent with the other containers)

**Pull request merged**